### PR TITLE
release-notes: add 5.2.1 release notes

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
release-notes: add 5.2.1 release notes

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-5.2.1.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+发版日期：2021 年 9 月 x 日
+
+TiDB 版本：5.2.1
+
+## Bug 修复
+
++ TiDB
+
+    - 修复在分区中下推聚合算子时，因浅拷贝 schema 列导致执行计划出错，进而导致执行结果出错的问题 [#27839](https://github.com/pingcap/tidb/pull/27839)
```
Comment (TomShawn) [inline releases/release-5.2.1.md:15]:
```suggestion
    - 修复在分区中下推聚合算子时，因浅拷贝 schema 列导致执行计划出错，进而导致执行结果出错的问题 [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
```

File: releases/release-5.2.1.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Bug 修复
+
++ TiDB
+
+    - 修复在分区中下推聚合算子时，因浅拷贝 schema 列导致执行计划出错，进而导致执行结果出错的问题 [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
+
++ TiKV
+
+    - 修复 Region 迁移时 Raftstore 模块出现死锁的问题 [#10909](https://github.com/tikv/tikv/issues/10909)
```
Comment (TomShawn) [inline releases/release-5.2.1.md:19]:
```suggestion
    - 修复 Region 迁移时 Raftstore 模块出现死锁导致 TiKV 不可用的问题 [#10909](https://github.com/tikv/tikv/issues/10909)
```

File: releases/release-5.2.1.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+发版日期：2021 年 9 月 x 日
+
+TiDB 版本：5.2.1
+
+## Bug 修复
+
++ TiDB
+
+    - 修复在分区中下推聚合算子时，因浅拷贝 schema 列导致执行计划出错，进而导致执行结果出错的问题 [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
```
Comment (time-and-fate) [inline releases/release-5.2.1.md:15]:
```suggestion
    - 修复在分区中下推聚合算子时，因浅拷贝 schema 列导致执行计划出错，进而导致执行时报错的问题 [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
```

**Review by time-and-fate — APPROVED** [review]
LGTM

File: releases/release-5.2.1.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## Bug 修复
+
++ TiDB
+
+    - 修复在分区中下推聚合算子时，因浅拷贝 schema 列导致执行计划出错，进而导致执行时报错的问题 [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
+
++ TiKV
+
+    - 修复 Region 迁移时 Raftstore 模块出现死锁导致 TiKV 不可用的问题 [#10909](https://github.com/tikv/tikv/issues/10909)
```
Comment (TomShawn) [inline releases/release-5.2.1.md:19]:
```suggestion
    - 修复 Region 迁移时 Raftstore 模块出现死锁导致 TiKV 不可用的问题。用户可通过关闭调度并重启出问题的 TiKV 来应对。[#10909](https://github.com/tikv/tikv/issues/10909)
```

File: releases/release-5.2.1.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## Bug 修复
+
++ TiDB
+
+    - 修复在分区中下推聚合算子时，因浅拷贝 schema 列导致执行计划出错，进而导致执行时报错的问题 [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
+
++ TiKV
+
+    - 修复 Region 迁移时 Raftstore 模块出现死锁导致 TiKV 不可用的问题。用户可通过关闭调度并重启出问题的 TiKV 来应对。[#10909](https://github.com/tikv/tikv/issues/10909)
```
Comment (TomShawn) [inline releases/release-5.2.1.md:19]:
```suggestion
    - 修复 Region 迁移时 Raftstore 模块出现死锁导致 TiKV 不可用的问题。用户可通过关闭调度并重启出问题的 TiKV 来临时应对。[#10909](https://github.com/tikv/tikv/issues/10909)
```

**Comment by TomShawn** [comment]
This notes are also approved by @nolouch and @alex-quan-001.

**Review by Liuxiaozhen12 — APPROVED** [review]

File: _index.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ aliases: ['/docs-cn/dev/']
 
 - [TiDB 简介](/overview.md)
 - [基本功能](/basic-features.md)
-- [TiDB 5.2 Release Notes](/releases/release-5.2.0.md)
+- [TiDB 5.2 Release Notes](/releases/release-5.2.1.md)
```
Comment (TomShawn) [inline _index.md:19]:
```suggestion
- [TiDB 5.2 Release Notes](/releases/release-5.2.0.md)
```

File: releases/release-5.2.1.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+title: TiDB 5.2.1 Release Notes
+---
+
+# TiDB 5.2.1 Release Notes
+
+发版日期：2021 年 9 月 x 日
```
Comment (TomShawn) [inline releases/release-5.2.1.md:7]:
```suggestion
发版日期：2021 年 9 月 9 日
```

**Pull request merged**